src/test/java/com/iexec/worker/compute/ComputeControllerTests.java

```diff
@@ -71,35 +74,77 @@ void sendExitCauseForComputeComputeStage() {
         when(workerpoolAuthorizationService.isSignedWithEnclaveChallenge(CHAIN_TASK_ID, AUTH_HEADER))
                 .thenReturn(true);
 
-        final ResponseEntity<?> response =
+        // Test single cause via deprecated endpoint
+        final ResponseEntity<?> singleResponse =
                 computeController.sendExitCauseForGivenComputeStage(AUTH_HEADER,
                         ComputeStage.PRE,
                         CHAIN_TASK_ID,
                         new ExitMessage(CAUSE));
-        Assertions.assertTrue(response.getStatusCode().is2xxSuccessful());
-        Assertions.assertEquals(HttpStatus.OK.value(), response.getStatusCode().value());
+        Assertions.assertTrue(singleResponse.getStatusCode().is2xxSuccessful());
+        Assertions.assertEquals(HttpStatus.OK.value(), singleResponse.getStatusCode().value());
+
+        // Test single cause via new bulk endpoint
+        final ResponseEntity<Void> bulkSingleResponse = computeController.sendExitCausesForGivenComputeStage(
+                AUTH_HEADER,
+                ComputeStage.POST,
+                CHAIN_TASK_ID,
+                Collections.singletonList(CAUSE)
+        );
+        Assertions.assertEquals(HttpStatus.OK.value(), bulkSingleResponse.getStatusCode().value());
+
+        // Test multiple causes via new bulk endpoint
+        List<ReplicateStatusCause> multipleCauses = Arrays.asList(
```

Comment:
For more concise imports, you can use
```suggestion
        List<ReplicateStatusCause> multipleCauses = List.of(
```

Reply:
Resolved in [a80e663](https://github.com/iExecBlockchainComputing/iexec-worker/pull/653/commits/a80e6637510785b3b500570dc702721b7a4f1d35)

Reply:
Not his occurrence, it is still visible in the **Files changed** view.
It is not marked as Outdated in the current view as well

Reply:
Resolved in [b39cedf](https://github.com/iExecBlockchainComputing/iexec-worker/pull/653/commits/b39cedf29e2273d24521dec561d52877bd438770)